src/components/virtual-gamepad-controls.css

```diff
@@ -47,9 +47,10 @@
   border-top-right-radius: 50%;
   border-bottom-right-radius: 50%;
   border-bottom-left-radius: 50%;
-  &__hidden {
-    background-color: transparent;
-  }
+}
+
+:local(.hidden) {
```

Comment:
Was this just an unrelated fix or is something about this syntax not supported in `sass`?

Reply:
Something about the syntax not being supported in sass. Although I think it's the css-loader not the sass-loader, so I'm not sure what exactly was erroring but this fixed it.